### ISSUE TITLE
Missing documentation on supports

### ISSUE THREAD

**Issue by Beep6581** [SoftFever/OrcaSlicer#7536]
### OrcaSlicer Version

2.2.0

### Operating System (OS)

Linux

### OS Version

Manjaro

### How to reproduce

Hover over "Top/Bottom Z distance" in Support > Advanced. The tooltip points to this URL, but such a page does not exist:
https://github.com/SoftFever/OrcaSlicer/wiki/support

### Actual results

URL does not exist.

### Expected results

Existing documentation regarding supports.

**Comment by Beep6581** [comment]
Curb your bot. Issue still valid.

**Comment by ArsenicBismuth** [comment]
Yeah this happens to multiple links, all redirect to an old links.
